sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureCliCredentialBuilder.java

```diff
@@ -58,11 +58,11 @@ public AzureCliCredentialBuilder tenantId(String tenantId) {
 
     /**
      * Specifies a {@link Duration} timeout for calling the Azure CLI.
```

Comment:
The Javadoc description can add following "The timeout period is applied on Azure CLI command execution process invoked by the credential."
Applies to Developer CLI below too.